google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/startup/CloudToolsPluginInitializationComponent.java

```diff
@@ -61,38 +47,13 @@ public void initComponent() {
     CloudToolsPluginInfoService pluginInfoService = ServiceManager
         .getService(CloudToolsPluginInfoService.class);
 
-    if (pluginInfoService.shouldEnable(GctFeature.APPENGINE_FLEX)) {
-      initAppEngineSupport(pluginConfigurationService);
-    }
-
     if (pluginInfoService.shouldEnableErrorFeedbackReporting()) {
       initErrorReporting(pluginConfigurationService, pluginInfoService);
     }
 
     new ConflictingAppEnginePluginCheck().notifyIfConflicting();
   }
 
-  private void initAppEngineSupport(
-      CloudToolsPluginConfigurationService pluginConfigurationService) {
-    AppEngineCloudType appEngineCloudType = new AppEngineCloudType();
-    pluginConfigurationService.registerExtension(ServerType.EP_NAME, appEngineCloudType);
-    pluginConfigurationService.registerExtension(ConfigurationType.CONFIGURATION_TYPE_EP,
-        new DeployToServerConfigurationType(appEngineCloudType));
```

Comment:
Is this extension point also handled by the additions to plugin.xml? I see the line item for `AppEngineCloudType` there, but not `DeployToServerConfigurationType`.